easybuild/tools/filetools.py

```diff
@@ -260,75 +261,34 @@ def download_file(filename, url, path):
     basedir = os.path.dirname(path)
     mkdir(basedir, parents=True)
 
-    # internal function to report on download progress
-    def report(blocks_read, blocksize, filesize):
```

Comment:
Hi,

2 reasons:

* urllib2 doesn't provide anything like this reporthook.
* On line 316 is the only call to that function, as far as I could tell there's also no way to actually see the progress from the command line

I figured that having the progress bar seems to be obsolete, and after rewriting the function I didn't see any use for it. Also: Rebuilding the progress report would have been cumbersome (and I'm lazy)